### ISSUE TITLE
Integrate cairo1 compiler to protostar

### ISSUE THREAD

**Issue by MaksymilianDemitraszek** [software-mansion/protostar#1486]
We are still waiting for the starknet 0.11, we can start the integration with new compiler.

We have all infrastructure to implement cairo1 based test runner, for now without any starknet support, we just assume for now test do not contain any cheatcodes. We want be able to import cairo1 libraries (not contracts!) and test them. We want let user to provide testing cairo path.

We also can integrate it to build command, with support of cairo paths

It depends on #1433 
Feel free to divide it into separate tasks

**Comment by Arcticae** [comment]
Split this issue into 2 mentioned above. Closing in favor of #1492 and #1491
